templates/permutsearch/index.html.twig

```diff
@@ -0,0 +1,67 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('permutsearch') }}
+{% endblock %}
+
+{% block title %}
+    {{ parent() }}
+    - Permut Search
+{% endblock %}
+
+{% block body %}
+    <main class="p-md-5 p-2 permutsearch">
+        <header class="mb-5">
+            <h1 class="h1 text-center text-dark">Permut Search</h1>
+            <h2 class="h2 text-center text-dark">Retrouvez vos différentes permutations potentielles</h2>
+        </header>
+
+        <div class="px-lg-5 px-2">
+            {% for user in users %}
+                {% if user.isVisible == true %}
+                    <div class="permut-card mb-5">
+                        <div class="permut-header mb-3 d-flex flex-row align-items-center justify-content-between">
+                            <div class="d-flex flex-row align-items-center text-white">
+                                <img class="rounded-circle me-4" src="https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png" alt="avatar-user">
+                                <div class="title-card">
+                                    <h3 class="h2 mb-1 text-warning">{{ user.username }}</h3>
+                                    <h4 class="h3">
+                                        {% if user.registeredUser.rome is defined %}
+                                            {{ user.registeredUser.rome }}
```

Comment:
shouldn't this be "user.registeredUser.rome.name"?

Reply:
you approved the PR of Mael first, so yes now it should be